### PR TITLE
Bandwith meter spike + first buffering callback.

### PR DESCRIPTION
**Pull request by nofxincubus**
Does not include error modal
https://www.pivotaltracker.com/story/show/167693225

### CODE REVIEW

File: android/src/main/java/jp/manse/BrightcovePlayerView.java [hunk 1]
```diff
@@ -70,16 +84,55 @@ public BrightcovePlayerView(ThemedReactContext context, AttributeSet attrs) {
...
+        this.progressBar.setIndeterminateDrawable(drawable);
+        this.progressBar.setVisibility(VISIBLE);
+    }
+
+    private int getBitrate(){
+        SharedPreferences pref = getContext().getSharedPreferences(TAG, 0);
+        if (pref.contains(BITRATE)) {
+            return pref.getInt(BITRATE,Integer.MAX_VALUE);
+        } else {
+            return Integer.MAX_VALUE;
```
Comment (jmargolese) [inline android/src/main/java/jp/manse/BrightcovePlayerView.java]: I just updated the AC.  Thinking about it, it makes sense to start with 3,500,000 i.e. an intermediate value if we don't know anything about the available bandwidth.  BC will adapt up to max if that works

**Review by jmargolese — CHANGES_REQUESTED** [review]
Please change the initial default value per the comment otherwise, looks good

File: android/src/main/java/jp/manse/BrightcovePlayerView.java (hunk 1 above)
Comment (nofxincubus) [reply android/src/main/java/jp/manse/BrightcovePlayerView.java]: Yep fixed

**Review by jmargolese — APPROVED** [review]
LGTM

**Pull request merged**